### DISCUSSION TITLE
Crypto.org Chain Dev Update #14

### DISCUSSION THREAD

**Discussion by aw126** [crypto-org-chain/chain-main#426] [Announcements]
![DevUpdate14_Blog](https://user-images.githubusercontent.com/78806365/112214495-a4056f00-8bf5-11eb-8902-9b52945314f4.gif)

Since our last update, the Crypto.org Chain team has been working on upgrading our chain and wallet application, integrating the technology offered by our partners, and preparing the Crypto.org Chain Mainnet launch. We have conducted internal dry runs, tested and improved the network in preparation for the upcoming Mainnet release.

Let’s dive into some details about what we have accomplished so far.

### Crypto.org Chain Roadmap in Summary
The upcoming roadmap for Crypto.org Chain is available [here](https://blog.crypto.com/crypto-org-chain-intergalactic-roadmap/), and there will be five major releases coming to Crypto.org Chain following the Mainnet launch!

- Non-Fungible Tokens (NFT) will be supported on-chain, and thereby users will be able to mint, trade, and transfer tokens on Crypto.org Chain.
- We plan to take advantage of the existing work of the Inter-Blockchain Communication (IBC) protocol to bridge Crypto.org Chain with Ethereum protocols, and onboard Ethereum developers as well as great projects to the Crypto.org Chain ecosystem.
- One of the key tasks on our list is to work closely with several leading protocols to utilize possible services from oracles.
- DEX integrate Automated Market Making (AMM) will be supported.
- Also, we will further conduct smart contract deployment to unload different smart contract standards.

### Partnership with Bison Trails
As previously [announced](https://blog.crypto.com/crypto-com-enlists-bison-trails-to-provide-validator-node-infrastructure-for-the-crypto-com-chain-testnet-and-upcoming-mainnet/), Crypto.org is partnering and collaborating with Bison Trails to enable high-efficient transaction block signing as well as the support of popular protocols. Node infrastructure clusters operated by Bison Trails are built on a multi-cloud foundation at various geographical regions. They enable Bison Trails to offer an easy-to-use platform that simplifies the validating process for validators to run nodes in a secure way.

With such support from our partner, validators are able to sign transaction blocks with low latency and high throughput on the Crypto.org Chain, and one of our internal dry runs started early this month is involved with Bison Trails hosted nodes. The dry run went as smooth as expected. Right after that, we have been working closely with Bison Trails in preparation for the Mainnet launch. Bison Trails is going to help us host our seed nodes to strengthen the Crypto.org Chain network in the initial stage.

### Partnership with Alchemy/Integration with Alchemy’s API
Alchemy’s Supernode is the most widely used Ethereum API with supercharged reliability, scalability, and data accuracy. Crypto.org is [partnering with Alchemy](https://blog.crypto.com/crypto-com-partners-with-alchemy-for-the-crypto-com-chain/) to provide a comprehensive suite of developer tools and services to help partners of Crypto.org Chain build great applications more efficiently.

We have implemented Alchemy API as a service to leverage our solutions. The integration with Alchemy APIs will enable a robust chain of data retrieval, transaction broadcasting, and chain monitoring experience to the community, thereby enhancing speed and security.

Our partnership will increase the accessibility of the Crypto.org Chain and reduce operational and maintenance costs for developers. We look forward to having more sparks with Alchemy to help build leading industrial Chain solutions.

### Introduction of an “Awesome” Community
To enhance our ecosystem and encourage engagement among the community, we are excited to introduce the “Awesome” project – where community members can share and showcase their Crypto.org Chain-based projects. The site provides details of libraries and clients’ repositories, wallet and ledger applications, indexing service, validator historical performance data et cetera. We encourage users to work together, help build this community-driven repository, and strengthen our Crypto.org Chain ecosystem. Check out the [Awesome community page](https://github.com/crypto-org-chain/awesome) and find out how you can start building!

### Mainnet Setup Instruction
March 25 is the scheduled launch date for Crypto.org Chain Mainnet! In the past few months, our team has successfully completed the platform’s internal dry runs and has finalized all features and functionalities for our Mainnet. We look forward to your participation and have made the Mainnet setup guide [here](https://crypto.org/docs/getting-started/mainnet.html) for those of you who would like to participate.

### Updates on Croeseid Testnet
Our Croeseid Testnet has integrated the Inter-Blockchain Communication (IBC) protocol which facilitates cross-chain communication and coordination - after Cosmos’s release of Stargate, their newest SDK. The main milestone of Stargate is the introduction of Cosmos’s first IBC protocol. IBC performs the role of a bridge among different blockchains, and this network of interconnected blockchains facilitates message communications and feature exchanges between chains with different infrastructure design and consensus algorithms. This upgrade will unlock the huge potential and create new capabilities for future cross-chain implementations of Crypto.org Chain and other Cosmos-based chains.

The previous governance proposal to enable IBC ended on March 1st. Voters have been very active in the voting process, resulting in a high voting rate of 85%. We would like to say thank you to our voters and supporters.

### Desktop Wallet Beta v0.0.25 Release
In the last dev update, we announced the release of the first Crypto.org Chain Desktop Wallet beta version v0.0.19, where users can directly conduct staking operations and transactions on Croeseid Testnet on this user-friendly desktop wallet. Since then, our team has received great feedback, worked on new features, and fixed the issues. The desktop wallet is now enhanced with the ledger support function. For the users who would like to use their Ledger devices with Crypto.org Chain desktop wallet, check out the [step-by-step instruction](https://crypto.org/docs/wallets/ledger_desktop_wallet.html#ledger-hardware-wallets).

With these changes, Crypto.org Chain Desktop Wallet [v0.0.25](https://github.com/crypto-com/chain-desktop-wallet/releases/tag/v0.0.24) has been released recently. This desktop wallet is available on Mac OS, Ubuntu, and Windows. To download the latest version or view further details, please refer to our [release page](https://github.com/crypto-com/chain-desktop-wallet/releases).

### Crypto.org Chain Binary Upgrade
Since the last update, our team has actively put effort into optimizing our network and released a few upgrade versions where you can find details [here](https://github.com/crypto-org-chain/chain-main/releases). By now, we have built up a version based on the upstream release of Cosmos SDK 0.42.1. This version is compatible with Go v1.16.1 and fixed the Initialization issue of the bank keeper. A security patch based on the findings in Gaia and Cosmos SDK is also updated in this pack.

### What’s in the Horizon
Along with the mainnet launch and following key releases, we will be sharing more updates about our progress on the NFT ecosystem, new features on the mainnet, further plans to drive a fast-growing DeFi on our Chain, and more. We are very excited to start this journey with you!

If you would like to be part of this, participate in the network or share your thoughts, keep an eye on our [Discord](https://discord.gg/5JTk2ppsY3) for updates and reach out to us at chain@crypto.org.

**Comment by 100hp4ever** [comment]
Why no more dev update since march?

**Reply by aw126** [reply]
Hi, thanks for the question! Recently there have been a few major updates that require more space to explain, so we post those updates in [separate postings](https://blog.crypto.com/tag/mainnet/), and also keep on updating the [Crypto.org Chain doc](https://crypto.org/docs/getting-started/mainnet.html) and [GitHub](https://github.com/crypto-org-chain/chain-main). We will continue to make new dev updates as necessary in near future.
